src/components/IOUConfirmationList.js

```diff
@@ -382,5 +490,11 @@ export default compose(
         network: {
             key: ONYXKEYS.NETWORK,
         },
+        user: {
+            key: ONYXKEYS.USER,
+        },
+        payPalMeUsername: {
```

Comment:
NAB: Should we use `payPalMeUsername` if the nvp says `address`. Maybe it would be more intuitive to stick with the address and use `payPalMeAddress`. What do you think?